### PR TITLE
Changed two translations

### PR DESCRIPTION
**Pull request by ZodexNL**
This pull request changes two translation.

`auth.php` made a translation feel more natural to read.

`passwords.php` fixed a spelling error.

### CODE REVIEW

File: passwords.php [hunk 1]
```diff
@@ -17,6 +17,6 @@
     'sent' => 'We hebben u uw wachtwoord reset link gemaild!',
     'throttled' => 'Wacht a.u.b. voor het opnieuw te proberen.',
     'token' => 'Deze wachtwoord reset token is ongeldig.',
-    'user' => "We kunnen geen gebruiker met dat mail adres vinden.",
+    'user' => "We kunnen geen gebruiker met dat mailadres vinden.",
```
Comment (vHeemstra) [inline passwords.php]: e-mailadres

Comment (ZodexNL) [reply passwords.php]: Thanks for the heads up! @vHeemstra

**Review by ZodexNL — COMMENTED** [review]
Improvements

**Comment by ZodexNL** [comment]
@johnhout Maybe take a look at the open pull requests? There are a few open which bring good changes to this repo.

**Pull request merged**